examples/how-to-fix-text-provider.kts

```diff
@@ -0,0 +1,19 @@
+object : HowToFixTextProvider {
+    fun OrtIssue.matchesMessage(pattern: String): Boolean = pattern.toRegex().matches(message)
```

Comment:
Wouldn't it be useful to have this function as part of the interface? I assume most how-to-fix texts will be triggered by matching a message.

Reply:
I intentionally did not add it here yet. My plan was to implement a few custom how-to-fix messages first and then, when these helper functions matured add them to ORT.